es6/assignment.js

```diff
@@ -1,96 +1,97 @@
-import { PlanOutOpRandom } from "./ops/random";
 import { shallowCopy, forEach, hasKey } from "./lib/utils";
 
-class Assignment {
-  constructor(experimentSalt, overrides) {
-    if (!overrides) {
-      overrides = {};
+export default function provideAssignement(Random) {
```

Comment:
weird. not sure how that let's anything work unless the import/export stuff doesn't care about the name here